cosmo_tester/resources/blueprints/component/nesting_component.yaml

```diff
@@ -0,0 +1,53 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  Create a file in an arbitrary location on a pre-existing VM using a plugin.
```

Comment:
Not without the plugin you won't- can you update the description to say what this is actually for?